atm.js

```diff
@@ -1,21 +1,63 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var checkBalance = 0;
+var saveBalance = 0;
 
```

Comment:
I think going forward, we'll be persisting data. Potentially we could have a balance of 150, but with your application it would automatically update the balance from 0.